lib/ts/recipe/webauthn/recipeImplementation.ts

```diff
@@ -371,56 +391,63 @@ export default function getRecipeImplementation(
             }
 
             // We should have received a valid registration options response.
-            let registrationResponse: RegistrationResponseJSON;
-            try {
-                registrationResponse = await startRegistration({ optionsJSON: registrationOptions });
-            } catch (error: any) {
-                if (error.name === "InvalidStateError") {
-                    return { status: "AUTHENTICATOR_ALREADY_REGISTERED" };
-                }
-
-                throw error;
+            const registerUserResponse = await this.registerUser({ registrationOptions });
```

Comment:
same here. The name should indicate that this is about the credential and not the user.